.github/workflows/wiki_push.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Workflow to push changes from the "wiki" folder in the repository towards the actual wiki itself.
+#
+name: 'Push to wiki'
```

Comment:
You don't need quotes, yaml automatically understands its a string.

Reply:
Still have it for consistency.